application/src/main/resources/ecc.yml

```diff
@@ -20,6 +20,24 @@
 ##
 connection:
   cql:
+    ## Configuration to define datacenters and hosts for ecchronos
+    ## to connect to. The 'enabled' property, set to false by default,
+    ## indicates whether the "datacenter aware" functionality will be active.
+    ## When 'enabled' is true, the application will use the configurations
+    ## specified in 'datacenters', connecting to the listed hosts.
+    datacenterAware:
+      enabled: false
+      datacenters:
+      - name: datacenter1
+        hosts:
+        - host: 127.0.0.1
```

Comment:
Is IP number the best thing, since I guess the solution will kind of be "dynamic".

Reply:
As we discussed, we will minimize the use of hosts using the cassandra driver to access initial contacts points to get the rest of them, also, I will create a task to investigate the use of fixed dns resolution as declared hosts and hostnames.